.github/workflows/build-with-bal-test-native.yml

```diff
@@ -17,6 +17,14 @@ on:
         default: ''
   schedule:
     -   cron: '30 18 * * *'
+    pull_request:
+      branches:
+        - main
```

Comment:
```suggestion
        - master
```

Reply:
There is an indentation issue as well

Reply:
fixed in[6f62ca7](https://github.com/ballerina-platform/module-ballerina-os/pull/486/commits/6f62ca700bef61f82550a23f9a66eae0efd4154a)